### DISCUSSION TITLE
Anyone experience this strange indent on the menu popup?

### DISCUSSION THREAD

**Discussion by nyngwang** [folke/noice.nvim#287] [Q&A] [answered]
I have to type one letter to get rid of it. I don't like it.

https://user-images.githubusercontent.com/24765272/209911616-2686f8d8-2160-4fa0-9e0a-e057117a8916.mov

**Comment by folke** [comment, marked as answer]
don't have that. seems weird. maybe your nui settings?

**Reply by nyngwang** [reply]
You're right, I think this is the bug caused by `nui.nvim`. This doesn't happen now.
